### PR TITLE
French translation update

### PR DESCRIPTION
**Pull request by Omicron666**
# Description
French translation of missing strings
+fix escaped tags
+rewording

Resolves https://github.com/EventFahrplan/EventFahrplan/issues/170

### CODE REVIEW

**Review by johnjohndoe — CHANGES_REQUESTED** [review]
Thank you for working on the French translation. I executed a first review and pointed out a few issues to change.

Once you are done with your changes, feel free to reorganize the commits on your `french-translation` branch using commands like `git commit --amend` or `git rebase --interactive` and to update this pull request via `git push --force`.

In any case let me know if I can help.

**Comment by johnjohndoe** [comment]
@Omicron666 Please let me know if I can help with the pull request - but take your time if you are busy these days.

**Comment by johnjohndoe** [comment]
@Omicron666 If you do not have time to finish this, please let me know so I can offer the issue to somebody else.

**Comment by Omicron666** [comment]
> @Omicron666 If you do not have time to finish this, please let me know so I can offer the issue to somebody else.

@johnjohndoe yes, please do. not on it at the moment

**Comment by johnjohndoe** [comment]
@Omicron666 Happy November. :maple_leaf: If you happen to have time to pick up this pull request again I am very glad to assist you to get it finalized. No time pressure here!

**Comment by Omicron666** [comment]
> 
> 
> @Omicron666 Happy November. 🍁 If you happen to have time to pick up this pull request again then I will be very glad to assist you to get it finalized. No time pressure here!

on it now

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -115,11 +114,60 @@
...
-    <string name="schedule_parsing_error_generic">Erreur inconnue lors de l’analyse du programme</string>
+    <string name="schedule_parsing_error_generic">Erreur inconnue lors de la lecture du programme</string>
     <string name="schedule_updated">Le programme a été mis à jour.</string>
-    <string name="progress_processing_data">Traitement du programme …</string>
+    <string name="progress_processing_data">Traitement du programme  &#8230;</string>
     <string name="schedule_changes_dialog_updated_to_text">"Le programme a été mis à jour "</string>
     <string name="uptodate">Le programme est à jour.</string>
     <string name="dlg_err_failed_not_found">URL du programme invalide</string>
-    <string name="progress_loading_data">Obtention du programme …</string>
+    <string name="progress_loading_data">Récupération du programme  &#8230;</string>
```
[the remark this answers is not in the document]
Comment (Omicron666) [reply app/src/main/res/values-fr/strings.xml]: fixed for progress_processing_data too

**Review by johnjohndoe — APPROVED** [review]
Merci for taking time again and the latest changes! Looks good to me.

:heavy_check_mark: Successfully tested on Pixel 2, Android 10.

**Comment by johnjohndoe** [comment]
Happy new year @omicron321! I started managing translations for the EventFahrplan project at [Crowdin](https://crowdin.com/project/eventfahrplan). I like to invite you to join there if you like.

**Pull request merged**